### ISSUE TITLE
is it possible to display token logo

### ISSUE THREAD

**Issue by ZebroTech** [blockscout/blockscout#7886]
![image](https://github.com/blockscout/blockscout/assets/32578764/a5555eb9-6b9c-421c-9eee-a0f2ffbe769c)
Is there any way you can show logo for each token?

**Comment by sl1depengwyn** [comment]
Logos in blockscout are provided by Trustwallet, if you do not have logo there we can't show it at the moment

**Comment by ZebroTech** [comment]
> Logos in blockscout are provided by Trustwallet, if you do not have logo there we can't show it at the moment

okay got it.
Possible update in future to display token logo without having it on TrustWallet?
BTW, If token is on trust wallet what to configure in blockscout to show up logo on explorer.

**Comment by vbaranov** [comment]
> Possible update in future to display token logo without having it on TrustWallet?

Where do you suppose to get such information?

> BTW, If token is on trust wallet what to configure in blockscout to show up logo on explorer.

You should set `CHAIN_ID` and `DISPLAY_TOKEN_ICONS=true`

**Comment by dreadedhamish** [comment]
> > Possible update in future to display token logo without having it on TrustWallet?
> 
> Where do you suppose to get such information?

I'm on a chain that has zero icons listed by Trustwallet (Pulsechain). In this instance I created my own repository that matches the Trustwallet structure.  Given the strict requirments that need to be met for Trustwallet to consider a project, in addition to the cost involved, lots of good projects will never get listed. 
I came to this repo and commented on this issue becuase I'm looking for an environmental variable to  switch to out custom repo.
